### PR TITLE
Feature/ar 59 courses table row selection

### PR DESCRIPTION
**Pull request by yuvallevanoni**

### CODE REVIEW

File: src/components/Screen-v2/CoursesScreen/ProductionStatusActions.tsx [hunk 1]
```diff
@@ -1,18 +1,20 @@
 import React, { Dispatch, ReactElement, useState, useCallback, useMemo } from 'react';
```
Comment (yuvallevanoni) [inline src/components/Screen-v2/CoursesScreen/ProductionStatusActions.tsx:2]: Important note
for this PR i'm not using this component...
I created PublishCoursesButton that i'm using instead

File: src/components/Screen-v2/CoursesScreen/buttons/PublishCoursesButton.tsx [hunk 2]
```diff
@@ -0,0 +1,132 @@
+import React, { Dispatch, ReactElement, useState, useCallback, useMemo } from 'react';
+
```
Comment (yuvallevanoni) [inline src/components/Screen-v2/CoursesScreen/buttons/PublishCoursesButton.tsx:2]: This file is similar to the **ProductionStatusActions** file, but it only contains the publish logic, without the archive..
the archive should be in a drop down component

File: src/components/Screen-v2/CoursesScreen/style.module.scss [hunk 3]
```diff
@@ -1,239 +1,176 @@
 @import 'src/styles/mixins';
```
Comment (yuvallevanoni) [inline src/components/Screen-v2/CoursesScreen/style.module.scss:1]: Here I also remove all old classes from the old design.
so no more '-' inside the classes names

**Review by dvira — APPROVED** [review]
Looks great!
Approved!

**Pull request merged**